scripts/devops_tasks/common_tasks.py

```diff
@@ -492,3 +492,74 @@ def get_package_properties(setup_py_path):
     pkgName, version, _, requires = parse_setup(setup_py_path)
     is_new_sdk = pkgName in NEW_REQ_PACKAGES or any(map(lambda x: (parse_require(x)[0] in NEW_REQ_PACKAGES), requires))
     return pkgName, version, is_new_sdk, setup_py_path
+
+def is_track2_package(reqs):
+    for req in reqs:
+        # the package must be dependant on azure core or azure mgmt core
+        if req.startswith('azure-core') or req.startswith('azure-mgmt-core'):
+            return True
+    return False
+
+def get_track2_package_properties(setup_py_path):
```

Comment:
I think you can get away with adding just this and the function above. Then you can just use the existing functionality to build up a version of `get_all_track2_packages`

I will have a better alternative for you in draft PR hopefully later today.